docs/stacks-guide-gcp.md

```diff
@@ -421,4 +421,4 @@ where we left off.
 [gcp-ip-address]: https://cloud.google.com/compute/docs/ip-addresses/
 [gcp-connection]: https://cloud.google.com/vpc/docs/configure-private-services-access
 [resource class selection]: https://github.com/crossplaneio/crossplane/blob/master/design/one-pager-simple-class-selection.md
-[gcp-provider-guide]: cloud-providers/gcp/gcp-provider.md
\ No newline at end of file
+[gcp-provider-guide]: cloud-providers/gcp/gcp-provider.md
```

Comment:
Any ideas why this is showing a diff?

Reply:
No o: . Must have something to do with whitespace as this change disappears when I hide whitespace changes, but I'm not sure what exactly.